toc.py

```diff
@@ -210,7 +210,7 @@ def set_argparser(argparser):
         # TODO: maybe leave only the short option to forbid configurability in config file
         group.add_argument("-f", "--force", action="store_true",
                 help="try to update the page even if it was last saved in the same UTC day")
-        group.add_argument("--toc-languages", default="all", type=ws.config.argtype_comma_list_choices(["all"] + lang.get_internal_tags()),
+        group.add_argument("--toc-languages", default="all", nargs="*", choices=["all"] + lang.get_internal_tags(),
```

Comment:
This should be `nargs="+"`, otherwise argparse would accept even an empty list.

Reply:
Yes, already fixed.